Winleafs.Wpf/Api/Orchestrator.cs

```diff
@@ -131,10 +132,41 @@ public async Task ActivateEffect(string effectName, int brightness)
             }
         }
 
-        /// <summary>
-        /// Returns the list of custom effects for the device
-        /// </summary>
-        public List<ICustomEffect> GetCustomEffects()
+		/// <summary>
+		/// Executes custom effect command
+		/// Activates an effect by name and brightness. This can be a custom effect (e.g. screen mirror) or a effect available on the Nanoleaf device
+		/// First deactivates any custom effects before enabling the new effect
+		/// </summary>
+		public async Task ExecuteCustomEffectCommand(CustomEffectCommand customEffectCommand)
+		{
+			_logger.Info($"Orchestrator is executing a custom effect command for device {Device.IPAddress}");
+
+			try
+			{
+				var client = NanoleafClient.GetClientForDevice(Device);
+
+				//DO NOT change the order of disabling effects, then setting brightness and then enabling effects
```

Comment:
Yes, I wasn't sure whether I would need to call `await _customEffects.DeactivateAllEffects();` or not before sending a command. It seemed to work fine without it, but you probably know more about why this had to be done when changing effects and it may apply here as well?